specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/machineLearningServices.json

```diff
@@ -6003,6 +6204,395 @@
+    },
+    "LabelingJob": {
+      "description": "Definition of a labeling job.",
+      "type": "object",
+      "properties": {
+        "labelCategories": {
+          "description": "Label categories of the job.",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/LabelCategory"
+          }
+        },
+        "jobInstructions": {
+          "description": "Instructions for the job.",
+          "$ref": "#/definitions/LabelingJobInstructions"
+        },
+        "datasetConfiguration": {
+          "description": "Dataset configuration for the job.",
+          "$ref": "#/definitions/LabelingDatasetConfiguration"
+        },
+        "mlAssistConfiguration": {
+          "description": "Machine learning assisted configuration for the job.",
+          "$ref": "#/definitions/MLAssistConfiguration"
+        },
+        "labelingJobMediaProperties": {
+          "description": "Media specific properties in a labeling job.",
+          "$ref": "#/definitions/LabelingJobMediaProperties"
+        },
+        "projectId": {
+          "format": "uuid",
+          "description": "Internal id of the job(Previously called project).",
+          "type": "string",
+          "readOnly": true
+        },
+        "status": {
+          "description": "Status of the job.",
+          "$ref": "#/definitions/JobStatus",
+          "readOnly": true
+        },
+        "progressMetrics": {
+          "description": "Progress metrics of the job.",
+          "$ref": "#/definitions/ProgressMetrics",
+          "readOnly": true
+        },
+        "statusMessages": {
+          "description": "Status messages of the job.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/StatusMessage"
+          },
+          "readOnly": true
+        },
+        "createdTimeUtc": {
+          "format": "date-time",
+          "description": "Created time of the job in UTC timezone.",
+          "type": "string",
+          "readOnly": true
+        },
+        "id": {
+          "description": "Name and identifier for a job.",
+          "type": "string",
+          "readOnly": true
+        },
+        "properties": {
+          "description": "The job property dictionary. Properties can be added, but not removed or altered.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "tags": {
+          "description": "The job tag dictionary. Tags can be added, removed, and updated.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "LabelingJobResourceRequest": {
+      "description": "The parameters for creating or updating a labeling job.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/LabelingJob"
+        }
+      }
+    },
+    "LabelingJobResource": {
```

Comment:
I am not quite sure but do we need a `x-ms-azure-resource` extension like [this](https://github.com/Azure/azure-rest-api-specs/blob/44754afdd63df701bd699c49cc74017d9dd62b75/specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/machineLearningServices.json#L2937) here since this is an azure resource model?

Reply:
Yes, good catch. Added to the `LabelingJobResource`.